### PR TITLE
[WIP] Ensure right class of network manager

### PR DESCRIPTION
**Pull request by Ladas**
Ensure right class of network manager, rest of the code can be
moved to baseclass then.

### CODE REVIEW

**Comment by durandom** [comment]
@Fryguy what about this ~~Harakiri~~ Hakiri guy? How can we fix his concerns?

**Comment by Fryguy** [comment]
It looks like Hakiri just wants the version of Rails bumped, but shouldn't it be Rails 5?

**Comment by durandom** [comment]
@Ladas can you rebase this branch and push again? Maybe this triggers Hakiri to rescan.
Re-Opening the PR doesnt work :(

File: app/models/manageiq/providers/amazon/cloud_manager.rb [hunk 1]
```diff
@@ -47,6 +47,10 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   supports :regions
   supports :discovery
 
+  def ensure_network_manager
```
Comment (durandom) [inline app/models/manageiq/providers/amazon/cloud_manager.rb]: Hmm, where is this called from? 
Isnt the `ensure_managers`  method where this should be?

Comment (Ladas) [inline app/models/manageiq/providers/amazon/cloud_manager.rb]: @durandom right, that was being leveraged in the next PR https://github.com/ManageIQ/manageiq-providers-amazon/pull/20/files

but I am no longer sure if it needs to be divided. Too many dependencies for my brain :-)

**Comment by durandom** [comment]
@Ladas rebase to trigger CI to restart?

**Comment by durandom** [comment]
@Ladas can you have a look why this is failing?

**Comment by Ladas** [comment]
@bronaghs we should merge this in, on my TODO list, keep falling through the cracks :-D

**Comment by djberg96** [comment]
@Ladas What's the current status of this?

**Comment by Ladas** [comment]
@djberg96 it moved up, it's now on 1459th place in my TODO list. :-)

**Comment by bronaghs** [comment]
srsly? Let's close this @Ladas.

**Comment by Ladas** [comment]
@bronaghs I would like to get it in, but I will need to check what changed in the file, I just can't find time to do it. :-)

**Pull request closed without merging**